src/engine/srv.c

```diff
@@ -644,6 +647,37 @@ dss_xstream_free(struct dss_xstream *dx)
 	D_FREE(dx);
 }
 
+static void
+dss_mem_stats_init(struct mem_stats *stats, int xs_id)
+{
+	int rc;
+
+	rc = d_tm_add_metric(&stats->ms_total_usage, D_TM_STATS_GAUGE,
+			     "Total memory usage", "byte", "mem/total_mem/xs_%u", xs_id);
```

Comment:
Just the byte count, not the actual alloc count so there's no chance to monitor average size?